paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -24,15 +24,17 @@ data class Environment(
   val resDir: String,
   val assetsDir: String,
   val packageName: String,
-  val compileSdkVersion: Int)
+  val compileSdkVersion: Int,
+  val platformDataDir: String,
+)
 
 @Suppress("unused")
 fun androidHome() = System.getenv("ANDROID_SDK_ROOT")
     ?: System.getenv("ANDROID_HOME")
     ?: "${System.getProperty("user.home")}/Library/Android/sdk"
 
 fun detectEnvironment(): Environment {
-  checkInstalledJvm()
+//  checkInstalledJvm()
```

Comment:
Nit: delete?

Reply:
oops, good call out.  reverted, since that check should still occur!